cms/static/js/views/settings/main.js

```diff
@@ -43,6 +45,8 @@ var DetailsView = ValidatingView.extend({
 
         this.listenTo(this.model, 'invalid', this.handleValidationError);
         this.listenTo(this.model, 'change', this.showNotificationBar);
+        this.listenTo(this.model, 'change:instructor_info', this.showNotificationBar);
```

Comment:
Given the general handling of model change (line 47), are these two lines necessary?